fares-data-build-tool/fdbt-site
# fix: time restrictions no longer breaks if yes selected and no days provided

# Description

-   time restrictions no longer breaks if yes selected and no days provided

# Testing instructions

-   To test, go to the time restrictions page, select yes, and select no days, then submit the page. The page should re-render with appropriate error message.

# Type of change


# Checklist:
